modules/claims_api/lib/brd/brd.rb

```diff
@@ -22,14 +29,29 @@ def countries
     # as {id: <number> and description: <string>}
     def intake_sites
       client.get('intake-sites').body[:items]
+    rescue => e
+      detail = e.respond_to?(:original_body) ? e.original_body : e
+      log_outcome_for_claims_api(service, 'error', detail)
+
+      error_handler(e)
```

Comment:
Nitpick: a tidy up ticket could be to wrap these in a method and simplify the rescues a bit.